File: backend/Logica/Cuenta.cs
namespace barcito.Logica
{
  public class Cuenta
  {
    public int IdCuenta { get; set; }
    public string Nombre { get; set; } = null!;
    public DateTime Fecha { get; set; }
    public string IdDevice { get; set; } = null!;
    public bool Pagado { get; set; } = true;  
  }

}

File: frontend-adm/src/app/core/interfaces/cuenta.ts
export interface Cuenta {
  IdCuenta: number;
  Nombre: string;
  idDevice: string;
  Fecha: string;
  Pagado: boolean;
  Mesa: number;
}

export interface CuentaConTotales {
  cuenta: Cuenta;
  totales: {
    total: number;
  };
}
